registry/storage/blob_test.go

```diff
@@ -139,6 +140,61 @@ func TestSimpleBlobUpload(t *testing.T) {
 	if digest.NewDigest("sha256", h) != sha256Digest {
 		t.Fatalf("unexpected digest from uploaded layer: %q != %q", digest.NewDigest("sha256", h), sha256Digest)
 	}
+
+	// Delete a blob
+	err = bs.Delete(ctx, desc.Digest)
+	if err != nil {
+		t.Fatalf("Unexpected error deleting blob")
+	}
+
+	d, err := bs.Stat(ctx, desc.Digest)
+	if err == nil {
+		t.Fatalf("unexpected non-error stating deleted blob: %s", d)
+	}
+
+	switch err {
+	case distribution.ErrBlobUnknown:
+		break
+	default:
+		t.Errorf("Unexpected error type stat-ing deleted manifest: %s", reflect.ValueOf(err).Type())
```

Comment:
No need to use `reflect` here. Format with `%#v` and the type information will be available.